include/vcpkg/base/system.mac.h

```diff
@@ -0,0 +1,10 @@
+#include <vcpkg/base/optional.h>
+#include <vcpkg/base/stringview.h>
+
+#include <string>
+
+namespace vcpkg
+{
+    Optional<StringView> find_first_nonzero_mac(StringView sv);
```

Comment:
```suggestion
```

It seems this is not called outside of the source file. (Also remove optional/stringview includes)

Reply:
It is used in the test code :'(

Reply:
Ah, makes sense!